src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -71,198 +43,42 @@ Here's an example of logs for your host's UI. You can see logs in context of eve
-
-    **Example:**
-
-    ```
-    sudo cp /etc/newrelic-infra/logging.d/elasticsearch-log.yml.example /etc/newrelic-infra/logging.d/elasticsearch-log.yml
-    ```
-  </Collapser>
-
-  <Collapser
-    id="mysql-logs"
-    title="MySQL logs"
-  >
-    Copy (or rename) the `mysql-log.yml.example` file to `mysql-log.yml` to enable automatic MySQL error log parsing and forwarding to New Relic. No need to restart the agent.
-
-    **Example:**
-
-    ```
-    sudo cp /etc/newrelic-infra/logging.d/mysql-log.yml.example /etc/newrelic-infra/logging.d/mysql-log.yml
-    ```
-  </Collapser>
-
-  <Collapser
-    id="nginx-logs"
-    title="NGINX logs"
-  >
-    Copy (or rename) the `nginx-log.yml.example` file to `nginx-log.yml` to enable automatic NGINX access and error log parsing and forwarding to New Relic. No need to restart the agent.
-
-    **Example:**
-
-    ```
-    sudo cp /etc/newrelic-infra/logging.d/nginx-log.yml.example /etc/newrelic-infra/logging.d/nginx-log.yml
-    ```
-  </Collapser>
-
-  <Collapser
-    id="rabbitmq-logs"
-    title="Rabbitmq logs"
-  >
-    Copy (or rename) the `rabbitmq-log.yml.example` file to `rabbitmq-log.yml` to enable automatic Rabbitmq error log parsing and forwarding to New Relic. No need to restart the agent.
-
-    **Example:**
-
-    ```
-    sudo cp /etc/newrelic-infra/logging.d/rabbitmq-log.yml.example /etc/newrelic-infra/logging.d/rabbitmq-log.yml
-    ```
-  </Collapser>
-
-  <Collapser
-    id="redis-logs"
-    title="Redis logs"
+<Collapser
+    className="freq-link"
+    id="system"
+    title="Review system requirements"
   >
-    Copy (or rename) the `redis-log.yml.example` file to `redis-log.yml` to enable automatic Redis error log parsing and forwarding to New Relic. No need to restart the agent.
-
-    **Example:**
 
-    ```
-    sudo cp /etc/newrelic-infra/logging.d/redis-log.yml.example /etc/newrelic-infra/logging.d/redis-log.yml
-    ```
-  </Collapser>
-</CollapserGroup>
-
-## System requirements [#requirements]
-
-To use the log forwarder of the infrastructure agent, make sure you meet the following requirements:
-
-* Infrastructure agent version 1.11.4 or higher.
+* Infrastructure agent version 1.11.4 or higher
 * [Fluent Bit](https://fluentbit.io/). The infrastructure agent already installs the latest version for you. To update or downgrade it to a specfic version, refer to the [Fluent Bit installation](#install-fb-version) procedures.
-* OpenSSL library 1.1.0 or higher is required by the infrastructure agent starting from version 1.16.4.
+* OpenSSL library 1.1.0 or higher
 * Built-in support for ARM64 architecture on Linux systems (for example, AWS Graviton architecture) added in infrastructure agent [1.20.6](https://github.com/newrelic/infrastructure-agent/releases/tag/1.20.6).
+* Amazon Linux 2
+* CentOS version 7 or higher
+* Debian version 9 ("Stretch") or higher. Version 11 is not supported.
```

Comment:
This isn't a big deal at all I don't think but the period makes the punctuation inconsistent. Just doing due diligence.